services/base-service/models/expert/model.js

```diff
@@ -5,6 +5,7 @@ const validate = require('../validate.js');
 
 const finApi = require('@ucd-lib/fin-api/lib/api.js');
 const config = require('../config');
+const Citation = require('../../spa/client/public/lib/utils/citation.js');
```

Comment:
@jrmerz you can see that @UcDust has linked to a shared library, both the server and client use.  Do you have a best practice for file organization in this case?